index.html

```diff
@@ -42,6 +42,8 @@
       <section id="main_content">
         <div class="button">
           <a href="http://ginnivan.blob.core.windows.net/carnac/Carnac.application">Download</a>
+          <p class="top">click to begin</p>
+          <p class="bottom">1.813 Byte .application</p>
```

Comment:
This file is just a bootstrapper, which will go and download the necessary files and install the application - it's really small and doesn't quite reflect reality. A more relevant number is the executable's size (3.20MB) - what about if we used that here?